cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -126,6 +136,43 @@ func httpServer(socketPath string, httpEndpoint string) {
 	klog.Fatal(http.ListenAndServe(httpEndpoint, mux))
 }
 
+func checkLiveRegistrationSocket(socketFile, csiDriverName string) error {
+	klog.V(1).Infof("Attempting to open a gRPC connection with: %q", socketFile)
+	cmm := metrics.NewCSIMetricsManagerForSidecar("")
```

Comment:
The manager should not be re-created for each health check. Please use the one from main.go

Reply:
I removed need for metrics manager. I think it was unncessary for this use case anyways.